### ISSUE TITLE
All actions should work with widgetId instead of bcName

### ISSUE THREAD

**Issue by Dergash** [tesler-platform/tesler-ui#16]
Currently many redux actions rely on `bcName` in their payload, despite them actually being widget-based - every widget is provided with widget meta, identified by widget's `name`.
As multiple widgets often share the same business component, this leads to confusion what exactly should be used in every particular case, `bcName` or widget's `name`, and often both are used, in that  case `bcName` redundant as it always can be calculated from widget's name.

**TL;DR**
All widget-based interactions should be identified by `widgetName`, and when `bcName` is required it should be calculated as follows:
```ts
const {widgetName} = action.payload
const state = store.getState()
const widget = state.view.widgets.find(item => item.name === widgetName)
const bcName = widget.bcName
```

As it is a major breaking change, it should be implemented in 2.0.0 release
